libs/data-types/src/lib/utilities/bootstrapDynamicTypes/bootstrapDynamicTypes.spec.ts

```diff
@@ -134,61 +146,32 @@ describe(`bootstrapDynamicTypes`, () => {
         const LONELY_UNION = 'LONELY_HEARTS_CLUB';
 
         class LonelyUnionUser {
-            @Union(LONELY_UNION, 'type', {
+            @UnionType(LONELY_UNION, {
                 label: 'foo',
                 description: `bar can be as lonely as foo, it's the loneliest var in the village hoo`,
             })
             foo: unknown;
         }
 
-        it(`should throw`, () => {
-            const act = () =>
-                bootstrapDynamicTypes([Whatsit, Widget, FactoryRoom, Undecorated, LonelyUnionUser]);
-
-            expect(act).toThrow();
-        });
-    });
-
-    describe(`when the same union has been registered twice with inconsistent discriminant paths`, () => {
-        const DUPLICATED_UNION = 'DUPLICATED_UNION';
-
-        class Woo {
-            @Union(DUPLICATED_UNION, 'type', buildOptions('foo'))
-            foo: unknown;
-        }
-
-        class Boo {
-            @Union(DUPLICATED_UNION, 'inconsistentDiscriminantFieldName', buildOptions('baz'))
-            baz: unknown;
-        }
-
-        @UnionMember(DUPLICATED_UNION, 'A')
-        class UnionMemberA {
-            inconsistentDiscriminantFieldName = 'A';
-        }
-
-        @UnionMember(DUPLICATED_UNION, 'B')
-        class UnionMemberB {
-            type = 'B';
-        }
-
         it(`should throw`, () => {
             const act = () =>
                 bootstrapDynamicTypes([
+                    ThingUnion,
                     Whatsit,
                     Widget,
                     FactoryRoom,
                     Undecorated,
-                    Woo,
-                    Boo,
-                    UnionMemberA,
-                    UnionMemberB,
+                    LonelyUnionUser,
                 ]);
 
             expect(act).toThrow();
         });
     });
 
+    describe(`when two unions have been defined with the same name`, () => {
```

Comment:
I presume this would trigger an error

Reply:
Yes. I made the description in the .todo(...) call more specific. Note that we will never use the lib this way, so I didn't worry too much about this particular test coverage.

Reply:
But I went ahead and added it now for good measure.